src/ui/sass/modules/_Results.scss

```diff
@@ -72,6 +72,72 @@ $results-cards-margin: calc(var(--yxt-base-spacing) / 2) !default;
     font-size: var(--yxt-results-title-bar-icon-size);
   }
 
+  &-viewAllLink
```

Comment:
this doesn't exist yet right? another PR?

Reply:
it's being used in resultssectionheader.hbs, I think I deleted resultssectionheader.hbs in #569 but then we brought it back at some point